deploy/README.md

```diff
@@ -351,8 +351,49 @@ helm repo update \
    && helm install stable/fluent-bit --name fluent-bit --namespace sumologic -f fluent-bit-overrides.yaml
 ```
 
+## Step 4: Deploy Falco
+
+In this step, you will deploy Falco to detect anomalous activity and capture Kubernetes Audit Events.
+
+__NOTE__ Falco needs privileged container access to insert its kernel module to process events for system calls.
+
+Download the file `falco-overrides.yaml` from GitHub:
+
+```sh
+curl -LJO https://raw.githubusercontent.com/SumoLogic/sumologic-kubernetes-collection/master/deploy/helm/falco-overrides.yaml
+```
+
+Install `falco` using Helm:
+
+```sh
+helm repo update \
+   && helm install stable/falco --name falco --namespace sumologic -f falco-overrides.yaml
+```
+
+__NOTE__ `Google Kubernetes Engine (GKE)` uses Container-Optimized OS (COS) as the default operating system for its worker node pools. COS is a security-enhanced operating system that limits access to certain parts of the underlying OS. Because of this security constraint, Falco cannot insert its kernel module to process events for system calls. However, COS provides the ability to leverage eBPF (extended Berkeley Packet Filter) to supply the stream of system calls to the Falco engine. eBPF is currently supported only on GKE and COS.
```

Comment:
Might be good to put a link to the official Falco docs here as well so customers can get more information.

Reply:
Will do.

Reply:
done